authorization/invitation/service/invitation_service.go

```diff
@@ -44,7 +46,7 @@ func NewInvitationService(context servicecontext.ServiceContext, config Invitati
 // This method creates one record in the INVITATION table for each user in the invitations parameter.  Any roles that are issued
 // as part of a user's invitation are created in the INVITATION_ROLE table.
 // IMPORTANT: This is a transactional method, which manages its own transaction/s internally
-func (s *invitationServiceImpl) Issue(ctx context.Context, issuingUserId uuid.UUID, inviteTo string, invitations []invitation.Invitation) error {
+func (s *invitationServiceImpl) Issue(ctx context.Context, issuingUserId uuid.UUID, inviteTo string, redirectURL *app.RedirectURL, invitations []invitation.Invitation) error {
```

Comment:
at the service level, we should have well defined arguments. So I would suggest that the block on lines 155-164 moves to the calling controller method. In other words, I would prefer not having method/function parameters of type `app.*`  (ie, generated by Goa) at the service level.